platform/tests/TESTS/mbed_platform/stats_cpu/main.cpp

```diff
@@ -28,7 +28,9 @@
 
 using namespace utest::v1;
 
+#ifdef LED1
```

Comment:
I feel this is not the right place to test LEDs.
Do you think we could remove it?

Reply:
Let's ignore this. The original code already uses LED1.
The proposed enhancement looks to ensure the code can run if LED1 isn't defined.